### DISCUSSION TITLE
Error pinging Docker server

### DISCUSSION THREAD

**Discussion by mohammedyusha** [coder/coder#4648] [General] [answered]
I have deployed Coder on AWS EC2(ubuntu) and was planning to create a code-server workspace. While doing that, I got an error. I have installed docker and also added my user to docker group. But the error still persists.

![WhatsApp Image 2022-10-19 at 09 24 41](https://user-images.githubusercontent.com/113102891/196594878-2a64e7dc-0a23-472c-b81c-98badb763959.jpeg)

**Comment by jsjoeio** [comment]
@bpmct any ideas on this one?

**Comment by matifali** [comment, marked as answer]
Please check
https://coder.com/docs/v2/latest/platforms/docker#permission-denied-while-trying-to-connect-to-the-docker-daemon-socket
